[2007-11-25 06:08:16,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:16,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 34539)
[2007-11-25 06:08:16,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:08:16,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:08:16,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:16,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 34540)
[2007-11-25 06:08:16,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:08:16,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:08:16,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:16,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 34541)
[2007-11-25 06:08:16,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:08:16,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:08:16,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:16,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 34542)
[2007-11-25 06:08:16,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:08:16,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:08:16,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:16,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 34543)
[2007-11-25 06:08:16,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:08:16,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:08:16,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:16,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 34544)
[2007-11-25 06:08:16,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:08:16,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:08:16,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:16,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 34545)
[2007-11-25 06:08:16,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:08:16,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:08:17,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:17,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 34546)
[2007-11-25 06:08:17,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:08:17,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:08:17,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:17,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 34547)
[2007-11-25 06:08:17,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:08:17,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:08:17,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:17,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 34548)
[2007-11-25 06:08:17,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:08:17,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:08:17,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:17,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 34549)
[2007-11-25 06:08:17,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:08:17,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:08:18,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:18,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 34550)
[2007-11-25 06:08:18,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:08:18,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:08:24,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:24,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 34551)
[2007-11-25 06:08:24,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:08:24,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:08:25,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:25,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 34552)
[2007-11-25 06:08:25,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:08:25,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:08:25,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:25,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 34553)
[2007-11-25 06:08:25,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:08:25,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:08:25,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:25,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 34554)
[2007-11-25 06:08:25,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:08:25,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:08:26,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:26,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 34555)
[2007-11-25 06:08:26,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:08:26,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:08:48,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:48,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 53744)
[2007-11-25 06:08:48,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 38, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 38)
[2007-11-25 06:08:48,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 38
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 38, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 38)

[2007-11-25 06:08:49,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:49,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 53745)
[2007-11-25 06:08:49,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 38, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 38)
[2007-11-25 06:08:49,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 38
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 38, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 38)

[2007-11-25 06:08:49,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:49,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 53746)
[2007-11-25 06:08:49,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 38, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 38)
[2007-11-25 06:08:49,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 38
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 38, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 38)

[2007-11-25 06:08:49,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:08:49,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 53747)
[2007-11-25 06:08:49,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 38, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 38)
[2007-11-25 06:08:49,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 38
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 38, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 38)

[2007-11-25 06:09:04,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:09:12,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:09:12,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 53755)
[2007-11-25 06:09:12,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:09:12,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:09:18,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:09:42,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:09:42,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 53758)
[2007-11-25 06:09:42,server,ERROR] It was: <type 'exceptions.SyntaxError'>; Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)
[2007-11-25 06:09:42,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 40
SyntaxError: Non-ASCII character '\xc3' in file /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py on line 40, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (server_config.py, line 40)

[2007-11-25 06:09:48,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:10:06,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:10:16,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:14:07,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:14:07,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 57464)
[2007-11-25 06:14:07,server,ERROR] It was: <type 'exceptions.SyntaxError'>; invalid syntax (server_config.py, line 47)
[2007-11-25 06:14:07,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 47
    %(server_ip, server_port, server_docroot, max_connections, max_bandwith)
    ^
SyntaxError: invalid syntax

[2007-11-25 06:14:09,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:14:09,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 57465)
[2007-11-25 06:14:09,server,ERROR] It was: <type 'exceptions.SyntaxError'>; invalid syntax (server_config.py, line 47)
[2007-11-25 06:14:09,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 47
    %(server_ip, server_port, server_docroot, max_connections, max_bandwith)
    ^
SyntaxError: invalid syntax

[2007-11-25 06:14:09,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:14:09,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 57466)
[2007-11-25 06:14:09,server,ERROR] It was: <type 'exceptions.SyntaxError'>; invalid syntax (server_config.py, line 47)
[2007-11-25 06:14:09,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 47
    %(server_ip, server_port, server_docroot, max_connections, max_bandwith)
    ^
SyntaxError: invalid syntax

[2007-11-25 06:14:10,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:14:10,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 57467)
[2007-11-25 06:14:10,server,ERROR] It was: <type 'exceptions.SyntaxError'>; invalid syntax (server_config.py, line 47)
[2007-11-25 06:14:10,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 47
    %(server_ip, server_port, server_docroot, max_connections, max_bandwith)
    ^
SyntaxError: invalid syntax

[2007-11-25 06:14:11,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:14:11,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 57468)
[2007-11-25 06:14:11,server,ERROR] It was: <type 'exceptions.SyntaxError'>; invalid syntax (server_config.py, line 47)
[2007-11-25 06:14:11,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 47
    %(server_ip, server_port, server_docroot, max_connections, max_bandwith)
    ^
SyntaxError: invalid syntax

[2007-11-25 06:14:24,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:14:24,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 57469)
[2007-11-25 06:14:24,server,ERROR] It was: <type 'exceptions.SyntaxError'>; invalid syntax (server_config.py, line 46)
[2007-11-25 06:14:24,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 46
    %(server_ip, server_port, server_docroot, max_connections, max_bandwith)
    ^
SyntaxError: invalid syntax

[2007-11-25 06:14:29,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:14:29,server,ERROR] Exception happened during processing of request from ('127.0.1.1', 57470)
[2007-11-25 06:14:29,server,ERROR] It was: <type 'exceptions.SyntaxError'>; invalid syntax (server_config.py, line 46)
[2007-11-25 06:14:29,server,ERROR] Traceback: Traceback (most recent call last):
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 1021, in process_request_Thread
    self.RequestHandlerClass(request, client_address, self)
  File "SocketServer.py", line 522, in __init__
    self.handle()
  File "BaseHTTPServer.py", line 316, in handle
    self.handle_one_request()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 171, in handle_one_request
    method()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 180, in do_GET
    f=self.do_GETorHEAD()
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/server.py", line 300, in do_GETorHEAD
    return web.do_GET(self, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 275, in do_GET
    return pageprocessor.do_GET(passthroughRequest, passthroughResponse, timeoutpage=timeout)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 320, in do_GET
    return self.webapp.run_snakelet(self.handler, self.url, self.pathinfo, self.args, passthroughRequest, passthroughResponse)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/snakeserver/webapp.py", line 513, in run_snakelet
    reload(module)
  File "/home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.py", line 46
    %(server_ip, server_port, server_docroot, max_connections, max_bandwith)
    ^
SyntaxError: invalid syntax

[2007-11-25 06:14:40,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:14:50,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:19:56,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:19:56,webapp,ERROR] PAGE '/streapy_config/serverConfig.sn' threw exception: TypeError: 'DBConnect' object is not iterable
[2007-11-25 06:20:34,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:20:34,webapp,ERROR] PAGE '/streapy_config/serverConfig.sn' threw exception: TypeError: connect_db() takes no arguments (1 given)
[2007-11-25 06:20:35,webapp,ERROR] PAGE '/streapy_config/serverConfig.sn' threw exception: TypeError: connect_db() takes no arguments (1 given)
[2007-11-25 06:20:44,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:20:44,webapp,ERROR] PAGE '/streapy_config/serverConfig.sn' threw exception: TypeError: 'NoneType' object is not iterable
[2007-11-25 06:20:57,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:20:57,webapp,ERROR] PAGE '/streapy_config/serverConfig.sn' threw exception: NameError: global name 'config' is not defined
[2007-11-25 06:21:46,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:25:29,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:27:44,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:27:44,YpageEngine,DEBUG] parsing ypage "server_config.y"... 
[2007-11-25 06:27:44,YpageEngine,DEBUG] compiling...
[2007-11-25 06:27:44,YpageEngine,DEBUG] done (10 ms, 4399 lines/sec)
[2007-11-25 06:29:06,YpageEngine,DEBUG] parsing ypage "server_config.y"... 
[2007-11-25 06:29:06,webapp,ERROR] Ypage-compiler error: script close expected @line 6 col 3 (file: /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.y)
[2007-11-25 06:29:26,YpageEngine,DEBUG] parsing ypage "server_config.y"... 
[2007-11-25 06:29:26,webapp,ERROR] Ypage-compiler error: syntax error, script end tag missing? @line 45 col -1 (file: /home/tobi/programming/streapy/snakelets_webserver/Snakelets-1.44/webapps/streapy_config/server_config.y)
[2007-11-25 06:29:50,YpageEngine,DEBUG] parsing ypage "server_config.y"... 
[2007-11-25 06:29:50,YpageEngine,DEBUG] compiling...
[2007-11-25 06:29:50,webapp,ERROR] PAGE '?' threw exception: IndentationError: expected an indented block (<Ypage_code_of_server_config>, line 14)
[2007-11-25 06:29:57,YpageEngine,DEBUG] parsing ypage "server_config.y"... 
[2007-11-25 06:29:57,YpageEngine,DEBUG] compiling...
[2007-11-25 06:29:57,YpageEngine,DEBUG] done (1 ms, 47000 lines/sec)
[2007-11-25 06:30:09,YpageEngine,DEBUG] parsing ypage "server_config.y"... 
[2007-11-25 06:30:09,YpageEngine,DEBUG] compiling...
[2007-11-25 06:30:09,YpageEngine,DEBUG] done (10 ms, 4699 lines/sec)
[2007-11-25 06:30:25,YpageEngine,DEBUG] parsing ypage "server_config.y"... 
[2007-11-25 06:30:25,YpageEngine,DEBUG] compiling...
[2007-11-25 06:30:25,YpageEngine,DEBUG] done (10 ms, 4699 lines/sec)
[2007-11-25 06:30:38,YpageEngine,DEBUG] parsing ypage "server_config.y"... 
[2007-11-25 06:30:38,YpageEngine,DEBUG] compiling...
[2007-11-25 06:30:38,YpageEngine,DEBUG] done (1 ms, 52000 lines/sec)
[2007-11-25 06:32:25,webapp,DEBUG] reloading snakelet module 'webapps.streapy_config.server_config'
[2007-11-25 06:33:28,YpageEngine,DEBUG] parsing ypage "config_template.y"... 
[2007-11-25 06:33:28,YpageEngine,DEBUG] compiling...
[2007-11-25 06:33:28,YpageEngine,DEBUG] done (10 ms, 4899 lines/sec)
[2007-11-25 16:46:22,server,INFO] ----------------------------------------
[2007-11-25 16:46:22,server,INFO] Server is starting! Snakelets 1.44
[2007-11-25 16:46:22,server,INFO] sendfile(2) is NOT available, compatible but slower file serving is used
[2007-11-25 16:46:22,server,INFO] Creating server on "tobi-desktop"
[2007-11-25 16:46:22,server,INFO] creating new server socket
[2007-11-25 16:46:22,plugin,INFO] Loading plugins...
[2007-11-25 16:46:22,server,WARNING] NOTICE: VirtualHost configuration is NOT enabled.
[2007-11-25 16:46:22,server,WARNING] Defaulting to loading all available web apps to the current host.
[2007-11-25 16:46:22,server,INFO] Processing virtual host 'tobi-desktop'
[2007-11-25 16:46:22,server,INFO] Loading webapp 'manage'
[2007-11-25 16:46:22,server,DEBUG] WEBAPP manage (1 snakelets )
[2007-11-25 16:46:22,server,DEBUG]  name = Server Management
[2007-11-25 16:46:22,server,DEBUG]   url = /manage/
[2007-11-25 16:46:22,webapp,INFO] Precompiling all Ypages...
[2007-11-25 16:46:22,YpageEngine,DEBUG] parsing ypage "access_disp.y"... 
[2007-11-25 16:46:22,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:22,YpageEngine,DEBUG] done (1 ms, 25000 lines/sec)
[2007-11-25 16:46:22,YpageEngine,DEBUG] parsing ypage "manage_template.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (19 ms, 550 lines/sec)
[2007-11-25 16:46:23,YpageEngine,DEBUG] parsing ypage "webapps.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (10 ms, 9599 lines/sec)
[2007-11-25 16:46:23,YpageEngine,DEBUG] parsing ypage "serverinfo.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (9 ms, 8700 lines/sec)
[2007-11-25 16:46:23,YpageEngine,DEBUG] parsing ypage "webappinfo.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (20 ms, 6099 lines/sec)
[2007-11-25 16:46:23,YpageEngine,DEBUG] parsing ypage "sessions.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (9 ms, 5600 lines/sec)
[2007-11-25 16:46:23,YpageEngine,DEBUG] parsing ypage "index.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (1 ms, 3000 lines/sec)
[2007-11-25 16:46:23,YpageEngine,DEBUG] parsing ypage "login.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (1 ms, 36000 lines/sec)
[2007-11-25 16:46:23,YpageEngine,DEBUG] parsing ypage "access_log.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (10 ms, 5599 lines/sec)
[2007-11-25 16:46:23,YpageEngine,DEBUG] parsing ypage "menu.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (10 ms, 5799 lines/sec)
[2007-11-25 16:46:23,server,INFO] Got webapp: [WebApp 'Server Management' on vhost 'tobi-desktop', urlprefix=/manage/]
[2007-11-25 16:46:23,server,INFO] Loading webapp 'account'
[2007-11-25 16:46:23,server,DEBUG] WEBAPP account (1 snakelets )
[2007-11-25 16:46:23,server,DEBUG]  name = Test Account/Login
[2007-11-25 16:46:23,server,DEBUG]   url = /account/
[2007-11-25 16:46:23,webapp,INFO] Precompiling all Ypages...
[2007-11-25 16:46:23,YpageEngine,DEBUG] parsing ypage "createprofile.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (9 ms, 4300 lines/sec)
[2007-11-25 16:46:23,YpageEngine,DEBUG] parsing ypage "profilecreated.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (10 ms, 2699 lines/sec)
[2007-11-25 16:46:23,YpageEngine,DEBUG] parsing ypage "logout.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (1 ms, 21000 lines/sec)
[2007-11-25 16:46:23,YpageEngine,DEBUG] parsing ypage "login.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (20 ms, 3599 lines/sec)
[2007-11-25 16:46:23,YpageEngine,DEBUG] parsing ypage "viewprofile.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (10 ms, 5699 lines/sec)
[2007-11-25 16:46:23,YpageEngine,DEBUG] parsing ypage "loggedin.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (1 ms, 20000 lines/sec)
[2007-11-25 16:46:23,server,INFO] Got webapp: [WebApp 'Test Account/Login' on vhost 'tobi-desktop', urlprefix=/account/]
[2007-11-25 16:46:23,server,INFO] Loading webapp 'shop'
[2007-11-25 16:46:23,server,DEBUG] WEBAPP shop (1 snakelets )
[2007-11-25 16:46:23,server,DEBUG]  name = E-shop demo
[2007-11-25 16:46:23,server,DEBUG]   url = /shop/
[2007-11-25 16:46:23,webapp,INFO] Precompiling all Ypages...
[2007-11-25 16:46:23,YpageEngine,DEBUG] parsing ypage "checkout.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (9 ms, 5900 lines/sec)
[2007-11-25 16:46:23,YpageEngine,DEBUG] parsing ypage "exit.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (10 ms, 3299 lines/sec)
[2007-11-25 16:46:23,YpageEngine,DEBUG] parsing ypage "products.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (1 ms, 29000 lines/sec)
[2007-11-25 16:46:23,YpageEngine,DEBUG] parsing ypage "cart.y"... 
[2007-11-25 16:46:23,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:23,YpageEngine,DEBUG] done (1 ms, 21000 lines/sec)
[2007-11-25 16:46:23,server,INFO] Got webapp: [WebApp 'E-shop demo' on vhost 'tobi-desktop', urlprefix=/shop/]
[2007-11-25 16:46:23,server,INFO] Loading webapp 'streapy_config'
[2007-11-25 16:46:24,server,DEBUG] WEBAPP streapy_config (1 snakelets )
[2007-11-25 16:46:24,server,DEBUG]  name = streaPy configuration Application
[2007-11-25 16:46:24,server,DEBUG]   url = /streapy_config/
[2007-11-25 16:46:24,webapp,INFO] Precompiling all Ypages...
[2007-11-25 16:46:24,YpageEngine,DEBUG] parsing ypage "index.y"... 
[2007-11-25 16:46:24,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:24,YpageEngine,DEBUG] done (10 ms, 499 lines/sec)
[2007-11-25 16:46:24,YpageEngine,DEBUG] parsing ypage "server_config.y"... 
[2007-11-25 16:46:24,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:24,YpageEngine,DEBUG] done (1 ms, 52000 lines/sec)
[2007-11-25 16:46:24,YpageEngine,DEBUG] parsing ypage "config_template.y"... 
[2007-11-25 16:46:24,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:24,YpageEngine,DEBUG] done (10 ms, 4899 lines/sec)
[2007-11-25 16:46:24,server,INFO] Got webapp: [WebApp 'streaPy configuration Application' on vhost 'tobi-desktop', urlprefix=/streapy_config/]
[2007-11-25 16:46:24,server,INFO] Loading webapp 'shared2'
[2007-11-25 16:46:24,server,DEBUG] WEBAPP shared2 (0 snakelets )
[2007-11-25 16:46:24,server,DEBUG]  name = Shared Session webapp 2
[2007-11-25 16:46:24,server,DEBUG]   url = /shared2/
[2007-11-25 16:46:24,server,WARNING] shared2 shares the session with other webapps. Possible security problem!
[2007-11-25 16:46:24,webapp,INFO] Precompiling all Ypages...
[2007-11-25 16:46:24,YpageEngine,DEBUG] parsing ypage "index.y"... 
[2007-11-25 16:46:24,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:24,YpageEngine,DEBUG] done (1 ms, 58000 lines/sec)
[2007-11-25 16:46:24,YpageEngine,DEBUG] parsing ypage "login.y"... 
[2007-11-25 16:46:24,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:24,YpageEngine,DEBUG] done (9 ms, 3100 lines/sec)
[2007-11-25 16:46:24,server,INFO] Got webapp: [WebApp 'Shared Session webapp 2' on vhost 'tobi-desktop', urlprefix=/shared2/]
[2007-11-25 16:46:24,server,INFO] Loading webapp 'shared1'
[2007-11-25 16:46:24,server,DEBUG] WEBAPP shared1 (0 snakelets )
[2007-11-25 16:46:24,server,DEBUG]  name = Shared Session webapp 1
[2007-11-25 16:46:24,server,DEBUG]   url = /shared1/
[2007-11-25 16:46:24,server,WARNING] shared1 shares the session with other webapps. Possible security problem!
[2007-11-25 16:46:24,webapp,INFO] Precompiling all Ypages...
[2007-11-25 16:46:24,YpageEngine,DEBUG] parsing ypage "index.y"... 
[2007-11-25 16:46:24,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:24,YpageEngine,DEBUG] done (10 ms, 5799 lines/sec)
[2007-11-25 16:46:24,YpageEngine,DEBUG] parsing ypage "login.y"... 
[2007-11-25 16:46:25,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:25,YpageEngine,DEBUG] done (1 ms, 31000 lines/sec)
[2007-11-25 16:46:25,server,INFO] Got webapp: [WebApp 'Shared Session webapp 1' on vhost 'tobi-desktop', urlprefix=/shared1/]
[2007-11-25 16:46:25,server,INFO] Loading webapp 'music'
[2007-11-25 16:46:25,server,DEBUG] WEBAPP music (3 snakelets )
[2007-11-25 16:46:25,server,DEBUG]  name = Music browser
[2007-11-25 16:46:25,server,DEBUG]   url = /music/
[2007-11-25 16:46:25,webapp,INFO] Precompiling all Ypages...
[2007-11-25 16:46:25,YpageEngine,DEBUG] parsing ypage "index.y"... 
[2007-11-25 16:46:25,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:25,YpageEngine,DEBUG] done (10 ms, 11899 lines/sec)
[2007-11-25 16:46:25,server,INFO] Got webapp: [WebApp 'Music browser' on vhost 'tobi-desktop', urlprefix=/music/]
[2007-11-25 16:46:25,server,INFO] Processing virtual host 'tobi-desktop' root webapp 'ROOT'
[2007-11-25 16:46:25,server,DEBUG] WEBAPP ROOT (0 snakelets )
[2007-11-25 16:46:25,server,DEBUG]  name = Web server root
[2007-11-25 16:46:25,server,DEBUG]   url = /
[2007-11-25 16:46:25,webapp,INFO] Precompiling all Ypages...
[2007-11-25 16:46:25,YpageEngine,DEBUG] parsing ypage "index.y"... 
[2007-11-25 16:46:25,YpageEngine,DEBUG] compiling...
[2007-11-25 16:46:25,YpageEngine,DEBUG] done (10 ms, 3999 lines/sec)
[2007-11-25 16:46:25,server,DEBUG] Got webapp: [WebApp 'Web server root' on vhost 'tobi-desktop', urlprefix=/]
[2007-11-25 16:46:25,server,INFO] Default virtual host: tobi-desktop:9080
[2007-11-25 16:46:25,server,INFO] 8 webapps registered.
[2007-11-25 16:46:25,server,INFO] Snakelets 1.44 using port 9080
[2007-11-25 16:53:49,server,WARNING] Server got a keyboard interrupt signal
[2007-11-25 16:53:49,server,WARNING] Server is shutting down.
[2007-11-25 16:53:49,server,WARNING] ----------------------------------------
[2007-11-25 16:53:49,server,INFO] Closing down webapps...
[2007-11-25 16:53:49,server,DEBUG] closing [WebApp 'Web server root' on vhost 'tobi-desktop', urlprefix=/]
[2007-11-25 16:53:49,server,DEBUG] closing [WebApp 'E-shop demo' on vhost 'tobi-desktop', urlprefix=/shop/]
[2007-11-25 16:53:49,server,DEBUG] closing [WebApp 'Test Account/Login' on vhost 'tobi-desktop', urlprefix=/account/]
[2007-11-25 16:53:49,server,DEBUG] closing [WebApp 'Shared Session webapp 1' on vhost 'tobi-desktop', urlprefix=/shared1/]
[2007-11-25 16:53:49,server,DEBUG] closing [WebApp 'Music browser' on vhost 'tobi-desktop', urlprefix=/music/]
[2007-11-25 16:53:49,server,DEBUG] closing [WebApp 'streaPy configuration Application' on vhost 'tobi-desktop', urlprefix=/streapy_config/]
[2007-11-25 16:53:49,server,DEBUG] closing [WebApp 'Server Management' on vhost 'tobi-desktop', urlprefix=/manage/]
[2007-11-25 16:53:49,server,DEBUG] closing [WebApp 'Shared Session webapp 2' on vhost 'tobi-desktop', urlprefix=/shared2/]
[2007-11-25 16:53:49,plugin,INFO] Stopping plugins...
[2007-11-25 16:53:49,server,INFO] Exiting.
[2007-11-25 16:53:51,server,INFO] ----------------------------------------
[2007-11-25 16:53:51,server,INFO] Server is starting! Snakelets 1.44
[2007-11-25 16:53:51,server,INFO] sendfile(2) is NOT available, compatible but slower file serving is used
[2007-11-25 16:53:51,server,INFO] Creating server on "tobi-desktop"
[2007-11-25 16:53:51,server,INFO] creating new server socket
[2007-11-25 16:53:51,plugin,INFO] Loading plugins...
[2007-11-25 16:53:51,server,WARNING] NOTICE: VirtualHost configuration is NOT enabled.
[2007-11-25 16:53:51,server,WARNING] Defaulting to loading all available web apps to the current host.
[2007-11-25 16:53:51,server,INFO] Processing virtual host 'tobi-desktop'
[2007-11-25 16:53:51,server,INFO] Loading webapp 'manage'
[2007-11-25 16:53:51,server,DEBUG] WEBAPP manage (1 snakelets )
[2007-11-25 16:53:51,server,DEBUG]  name = Server Management
[2007-11-25 16:53:51,server,DEBUG]   url = /manage/
[2007-11-25 16:53:51,webapp,INFO] Precompiling all Ypages...
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "access_disp.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (9 ms, 2500 lines/sec)
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "manage_template.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (10 ms, 1099 lines/sec)
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "webapps.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (9 ms, 9600 lines/sec)
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "serverinfo.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (9 ms, 8700 lines/sec)
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "webappinfo.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (20 ms, 6099 lines/sec)
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "sessions.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (9 ms, 5600 lines/sec)
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "index.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (1 ms, 3000 lines/sec)
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "login.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (10 ms, 3599 lines/sec)
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "access_log.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (10 ms, 5599 lines/sec)
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "menu.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (9 ms, 5800 lines/sec)
[2007-11-25 16:53:51,server,INFO] Got webapp: [WebApp 'Server Management' on vhost 'tobi-desktop', urlprefix=/manage/]
[2007-11-25 16:53:51,server,INFO] Loading webapp 'account'
[2007-11-25 16:53:51,server,DEBUG] WEBAPP account (1 snakelets )
[2007-11-25 16:53:51,server,DEBUG]  name = Test Account/Login
[2007-11-25 16:53:51,server,DEBUG]   url = /account/
[2007-11-25 16:53:51,webapp,INFO] Precompiling all Ypages...
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "createprofile.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (1 ms, 43000 lines/sec)
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "profilecreated.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (10 ms, 2699 lines/sec)
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "logout.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (1 ms, 21000 lines/sec)
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "login.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (10 ms, 7199 lines/sec)
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "viewprofile.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (9 ms, 5700 lines/sec)
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "loggedin.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (10 ms, 1999 lines/sec)
[2007-11-25 16:53:51,server,INFO] Got webapp: [WebApp 'Test Account/Login' on vhost 'tobi-desktop', urlprefix=/account/]
[2007-11-25 16:53:51,server,INFO] Loading webapp 'shop'
[2007-11-25 16:53:51,server,DEBUG] WEBAPP shop (1 snakelets )
[2007-11-25 16:53:51,server,DEBUG]  name = E-shop demo
[2007-11-25 16:53:51,server,DEBUG]   url = /shop/
[2007-11-25 16:53:51,webapp,INFO] Precompiling all Ypages...
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "checkout.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (1 ms, 59000 lines/sec)
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "exit.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (1 ms, 33000 lines/sec)
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "products.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (10 ms, 2899 lines/sec)
[2007-11-25 16:53:51,YpageEngine,DEBUG] parsing ypage "cart.y"... 
[2007-11-25 16:53:51,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:51,YpageEngine,DEBUG] done (9 ms, 2100 lines/sec)
[2007-11-25 16:53:51,server,INFO] Got webapp: [WebApp 'E-shop demo' on vhost 'tobi-desktop', urlprefix=/shop/]
[2007-11-25 16:53:51,server,INFO] Loading webapp 'streapy_config'
[2007-11-25 16:53:52,server,ERROR] Error creating webapp: 'module' object has no attribute 'ServerConfig'
[2007-11-25 16:53:52,server,ERROR] Server couldn't start due to errors.
[2007-11-25 16:53:53,server,INFO] ----------------------------------------
[2007-11-25 16:53:53,server,INFO] Server is starting! Snakelets 1.44
[2007-11-25 16:53:53,server,INFO] sendfile(2) is NOT available, compatible but slower file serving is used
[2007-11-25 16:53:53,server,INFO] Creating server on "tobi-desktop"
[2007-11-25 16:53:53,server,INFO] creating new server socket
[2007-11-25 16:53:53,plugin,INFO] Loading plugins...
[2007-11-25 16:53:53,server,WARNING] NOTICE: VirtualHost configuration is NOT enabled.
[2007-11-25 16:53:53,server,WARNING] Defaulting to loading all available web apps to the current host.
[2007-11-25 16:53:53,server,INFO] Processing virtual host 'tobi-desktop'
[2007-11-25 16:53:53,server,INFO] Loading webapp 'manage'
[2007-11-25 16:53:53,server,DEBUG] WEBAPP manage (1 snakelets )
[2007-11-25 16:53:53,server,DEBUG]  name = Server Management
[2007-11-25 16:53:53,server,DEBUG]   url = /manage/
[2007-11-25 16:53:53,webapp,INFO] Precompiling all Ypages...
[2007-11-25 16:53:53,YpageEngine,DEBUG] parsing ypage "access_disp.y"... 
[2007-11-25 16:53:53,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:53,YpageEngine,DEBUG] done (1 ms, 25000 lines/sec)
[2007-11-25 16:53:53,YpageEngine,DEBUG] parsing ypage "manage_template.y"... 
[2007-11-25 16:53:53,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:53,YpageEngine,DEBUG] done (9 ms, 1100 lines/sec)
[2007-11-25 16:53:53,YpageEngine,DEBUG] parsing ypage "webapps.y"... 
[2007-11-25 16:53:53,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:53,YpageEngine,DEBUG] done (9 ms, 9600 lines/sec)
[2007-11-25 16:53:53,YpageEngine,DEBUG] parsing ypage "serverinfo.y"... 
[2007-11-25 16:53:53,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:53,YpageEngine,DEBUG] done (20 ms, 4349 lines/sec)
[2007-11-25 16:53:53,YpageEngine,DEBUG] parsing ypage "webappinfo.y"... 
[2007-11-25 16:53:53,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:53,YpageEngine,DEBUG] done (20 ms, 6099 lines/sec)
[2007-11-25 16:53:53,YpageEngine,DEBUG] parsing ypage "sessions.y"... 
[2007-11-25 16:53:53,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:53,YpageEngine,DEBUG] done (1 ms, 56000 lines/sec)
[2007-11-25 16:53:53,YpageEngine,DEBUG] parsing ypage "index.y"... 
[2007-11-25 16:53:53,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:53,YpageEngine,DEBUG] done (1 ms, 3000 lines/sec)
[2007-11-25 16:53:53,YpageEngine,DEBUG] parsing ypage "login.y"... 
[2007-11-25 16:53:53,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:53,YpageEngine,DEBUG] done (10 ms, 3599 lines/sec)
[2007-11-25 16:53:53,YpageEngine,DEBUG] parsing ypage "access_log.y"... 
[2007-11-25 16:53:53,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:53,YpageEngine,DEBUG] done (9 ms, 5600 lines/sec)
[2007-11-25 16:53:53,YpageEngine,DEBUG] parsing ypage "menu.y"... 
[2007-11-25 16:53:53,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:53,YpageEngine,DEBUG] done (10 ms, 5799 lines/sec)
[2007-11-25 16:53:53,server,INFO] Got webapp: [WebApp 'Server Management' on vhost 'tobi-desktop', urlprefix=/manage/]
[2007-11-25 16:53:53,server,INFO] Loading webapp 'account'
[2007-11-25 16:53:53,server,DEBUG] WEBAPP account (1 snakelets )
[2007-11-25 16:53:53,server,DEBUG]  name = Test Account/Login
[2007-11-25 16:53:53,server,DEBUG]   url = /account/
[2007-11-25 16:53:53,webapp,INFO] Precompiling all Ypages...
[2007-11-25 16:53:53,YpageEngine,DEBUG] parsing ypage "createprofile.y"... 
[2007-11-25 16:53:53,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:53,YpageEngine,DEBUG] done (10 ms, 4299 lines/sec)
[2007-11-25 16:53:53,YpageEngine,DEBUG] parsing ypage "profilecreated.y"... 
[2007-11-25 16:53:53,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:53,YpageEngine,DEBUG] done (1 ms, 27000 lines/sec)
[2007-11-25 16:53:53,YpageEngine,DEBUG] parsing ypage "logout.y"... 
[2007-11-25 16:53:53,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:53,YpageEngine,DEBUG] done (1 ms, 21000 lines/sec)
[2007-11-25 16:53:53,YpageEngine,DEBUG] parsing ypage "login.y"... 
[2007-11-25 16:53:53,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:53,YpageEngine,DEBUG] done (9 ms, 7200 lines/sec)
[2007-11-25 16:53:53,YpageEngine,DEBUG] parsing ypage "viewprofile.y"... 
[2007-11-25 16:53:53,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:53,YpageEngine,DEBUG] done (10 ms, 5699 lines/sec)
[2007-11-25 16:53:53,YpageEngine,DEBUG] parsing ypage "loggedin.y"... 
[2007-11-25 16:53:53,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:53,YpageEngine,DEBUG] done (1 ms, 20000 lines/sec)
[2007-11-25 16:53:53,server,INFO] Got webapp: [WebApp 'Test Account/Login' on vhost 'tobi-desktop', urlprefix=/account/]
[2007-11-25 16:53:53,server,INFO] Loading webapp 'shop'
[2007-11-25 16:53:53,server,DEBUG] WEBAPP shop (1 snakelets )
[2007-11-25 16:53:53,server,DEBUG]  name = E-shop demo
[2007-11-25 16:53:53,server,DEBUG]   url = /shop/
[2007-11-25 16:53:53,webapp,INFO] Precompiling all Ypages...
[2007-11-25 16:53:53,YpageEngine,DEBUG] parsing ypage "checkout.y"... 
[2007-11-25 16:53:53,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:53,YpageEngine,DEBUG] done (10 ms, 5899 lines/sec)
[2007-11-25 16:53:53,YpageEngine,DEBUG] parsing ypage "exit.y"... 
[2007-11-25 16:53:54,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:54,YpageEngine,DEBUG] done (1 ms, 33000 lines/sec)
[2007-11-25 16:53:54,YpageEngine,DEBUG] parsing ypage "products.y"... 
[2007-11-25 16:53:54,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:54,YpageEngine,DEBUG] done (9 ms, 2900 lines/sec)
[2007-11-25 16:53:54,YpageEngine,DEBUG] parsing ypage "cart.y"... 
[2007-11-25 16:53:54,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:54,YpageEngine,DEBUG] done (1 ms, 21000 lines/sec)
[2007-11-25 16:53:54,server,INFO] Got webapp: [WebApp 'E-shop demo' on vhost 'tobi-desktop', urlprefix=/shop/]
[2007-11-25 16:53:54,server,INFO] Loading webapp 'streapy_config'
[2007-11-25 16:53:54,server,ERROR] Error creating webapp: 'module' object has no attribute 'ServerConfig'
[2007-11-25 16:53:54,server,ERROR] Server couldn't start due to errors.
[2007-11-25 16:53:55,server,INFO] ----------------------------------------
[2007-11-25 16:53:55,server,INFO] Server is starting! Snakelets 1.44
[2007-11-25 16:53:55,server,INFO] sendfile(2) is NOT available, compatible but slower file serving is used
[2007-11-25 16:53:55,server,INFO] Creating server on "tobi-desktop"
[2007-11-25 16:53:55,server,INFO] creating new server socket
[2007-11-25 16:53:55,plugin,INFO] Loading plugins...
[2007-11-25 16:53:55,server,WARNING] NOTICE: VirtualHost configuration is NOT enabled.
[2007-11-25 16:53:55,server,WARNING] Defaulting to loading all available web apps to the current host.
[2007-11-25 16:53:55,server,INFO] Processing virtual host 'tobi-desktop'
[2007-11-25 16:53:55,server,INFO] Loading webapp 'manage'
[2007-11-25 16:53:55,server,DEBUG] WEBAPP manage (1 snakelets )
[2007-11-25 16:53:55,server,DEBUG]  name = Server Management
[2007-11-25 16:53:55,server,DEBUG]   url = /manage/
[2007-11-25 16:53:55,webapp,INFO] Precompiling all Ypages...
[2007-11-25 16:53:55,YpageEngine,DEBUG] parsing ypage "access_disp.y"... 
[2007-11-25 16:53:55,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:55,YpageEngine,DEBUG] done (9 ms, 2500 lines/sec)
[2007-11-25 16:53:55,YpageEngine,DEBUG] parsing ypage "manage_template.y"... 
[2007-11-25 16:53:55,YpageEngine,DEBUG] compiling...
[2007-11-25 16:53:55,YpageEngine,DEBUG] done (1 ms, 11000 lines/sec)
